frontend/src/home/Home.tsx

```diff
@@ -266,6 +276,12 @@ class HomeComponent extends React.Component<Props> {
     ));
   }
 
+  setSchedule(schedule: Schedule) {
+    console.log(schedule);
+    const [dndschedule, counter] = convertToDNDSchedule(schedule, 0);
```

Comment:
We would actually need to `prereqify` the schedule before we set it here. Without this we won't be able to generate the pre-req/co-req warnings for the course objects in the schedule that came from the excel sheets.

To do this you could make use of the utility function `prereqifySchedule` in `scrapers/src/prereq_loader.ts`. You need to provide it with a `DataLoader` object. An example usage of this function is in `addPrereqsToSchedules` in the same file.

Reply:
Added Prerequefying schedule